README.md

```diff
@@ -15,16 +15,21 @@
 APILyzer: A simple API Analyzer
 ===============================
 
-View this plugin's documentation at: http://revelc.github.io/apilyzer-maven-plugin
+[APILyzer][2] is a maven plugin that detects illegal changes to a maven
```

Comment:
So, I kinda want the main link to the plugin documentation very prominent at the top. I'd like to move a lot of the more detailed information here to that site, keeping only the bare minimum here.

For now, I'm fine with the additional details, but I'd still like to keep the URL prominently displayed.